test/App/DocsSpec.hs

```diff
@@ -0,0 +1,40 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
I fought a bit against orphaned `MonadHttp` but not sure, how else to do without it.